### PR TITLE
chore: stronger verify tests for addresses

### PR DESCRIPTION
**Pull request by Justintime50**
# Description

Adds stronger address verify tests to ensure bools and arrays work.

# Testing

- Updates old tests to use bools
- Adds a new test to ensure that arrays still work

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: EasyPost/Address.cs [hunk 1]
```diff
@@ -92,8 +87,8 @@ public static async Task<AddressCollection> All(Dictionary<string, object>? para
         ///     * {"country", string}
         ///     * {"phone", string}
         ///     * {"email", string}
-        ///     * {"verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
-        ///     * {"strict_verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
+        ///     * {"verifications", List&lt;Any&gt;} Possible items are true, "delivery" and "zip4".
+        ///     * {"strict_verifications", List&lt;Any&gt;} Possible items are true, "delivery" and "zip4".
```
Comment (Justintime50) [inline EasyPost/Address.cs:91]: @nwithan8 can you help me update this docstring to say either a bool, string, or list of any?

File: EasyPost/Address.cs [hunk 2]
```diff
@@ -92,8 +87,8 @@ public static async Task<AddressCollection> All(Dictionary<string, object>? para
         ///     * {"country", string}
         ///     * {"phone", string}
         ///     * {"email", string}
-        ///     * {"verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
-        ///     * {"strict_verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
+        ///     * {"verifications", object} Possible items are true, "delivery" and "zip4".
+        ///     * {"strict_verifications", object} Possible items are true, "delivery" and "zip4".
```
Comment (Justintime50) [inline EasyPost/Address.cs:91]: Question: why object? It should be one of list, bool, str. Now people will think it must be an object of some undefined type?

**Review by Justintime50 — COMMENTED** [review]
Code looks good, one question @nwithan8

File: EasyPost/Address.cs (hunk 2 above)
Comment (nwithan8) [reply EasyPost/Address.cs:91]: There's no such thing as "one of multiple types" in a type-strict language 😉 The way to do that is say it's the base type `object`

Comment (Justintime50) [reply EasyPost/Address.cs:91]: Okie

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**